src/megameklab/com/MegaMekLab.java

```diff
@@ -26,160 +31,58 @@
 import java.util.Locale;
 
 import megamek.MegaMek;
+import megamek.common.Configuration;
 import megamek.common.QuirksHandler;
 import megamek.common.logging.DefaultMmLogger;
 import megamek.common.logging.LogConfig;
 import megamek.common.logging.LogLevel;
 import megamek.common.logging.MMLogger;
 import megamek.common.preference.PreferenceManager;
-import megameklab.com.ui.Mek.MainUI;
+import megamek.common.util.MegaMekFile;
+import megameklab.com.ui.StartupGUI;
 
 public class MegaMekLab {
     public static final String VERSION = "0.47.1-SNAPSHOT";
 
+    private static final String FILENAME_BT_CLASSIC_FONT = "btclassic/BTLogo_old.ttf"; //$NON-NLS-1$
+
     private static MMLogger logger = null;
 
     public static void main(String[] args) {
-        final String METHOD_NAME = "main(String[])";
-        
     	System.setProperty("apple.laf.useScreenMenuBar", "true");
         System.setProperty("com.apple.mrj.application.apple.menu.about.name","MegaMekLab");
-
-        String logFileName = "./logs/megameklablog.txt";
-        Locale.setDefault(Locale.US);
-
-        //Taharqa: I am not sure why this is here, so I am commenting it
-        //out for awhile because I suspect it might be responsible for the
-        //partial unit.cache problem in MHQ.
-        //new File("./data/mechfiles/units.cache").delete();
-
-        boolean logs = true;
-        boolean vehicle = false;
-        boolean battlearmor = false;
-
-        for (String arg : args) {
-            if (arg.equalsIgnoreCase("-vehicle")) {
-                vehicle = true;
-            }
-
-            if (arg.equalsIgnoreCase("-battlearmor")) {
-                battlearmor = true;
-            }
-
-            if (arg.equalsIgnoreCase("-nolog")) {
-                logs = false;
-            }
-        }
-
-        setupLogging(logs, logFileName);
-        showInfo();
-        
-        if (vehicle) {
-            Runtime runtime = Runtime.getRuntime();
-
-            getLogger().log(MegaMekLab.class, METHOD_NAME, LogLevel.INFO,
-                            "Memory Allocated [" +
-                            (runtime.maxMemory() / 1000) + "]");
-            // Need at least 200m to run MegaMekLab
-            if (runtime.maxMemory() < 200000000) {
```

Comment:
I think this is very old code that was used to brute force past a memory leak issue. I doubt it is still an issue and if so this does not seem like a very good way to handle it.